content/02.main-text.md

```diff
@@ -257,3 +263,25 @@ Manubot provides an ideal platform for perpetual reviews [@arxiv:1502.01329; @ta
 
 Our process represents an early step toward open massively collaborative reviews, and there are certainly aspects that can be improved.
 We invite the scientific community to adapt and build upon our experience and open software.
```

Comment:
Should this paragraph be deleted or integrated elsewhere?

Reply:
These two sentences were only one option for a closing.  Assuming we keep your new closing, we can remove them both.

The paragraph above starting with ` Open writing presents new opportunities for scholarly communication.` isn't really a limitation.  Should we integrate that somewhere else?